testsuite/src/it/java/io/quarkus/ts/startstop/ArtifactGeneratorTest.java

```diff
@@ -107,13 +107,13 @@
             "resteasy-jsonb",
             "scheduler",
             "spring-boot-properties",
-            "smallrye-reactive-messaging-amqp",
+            "smallrye-reactive-streams-operators",
             "spring-data-jpa",
             "spring-di",
             "spring-security",
             "spring-web",
             "undertow",
-            "undertow-websockets",
+            "websockets",
```

Comment:
Put this to `supportedExtensionsSubsetSetB`

Reply:
Done